### DISCUSSION TITLE
2010/01/11/%E5%9C%A8google-wave%E4%B8%8A%E8%AE%A8%E8%AE%BAlabview%E9%97%AE%E9%A2%98

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#288] [Announcements]
# 2010/01/11/%E5%9C%A8google-wave%E4%B8%8A%E8%AE%A8%E8%AE%BAlabview%E9%97%AE%E9%A2%98

如果还不了解 Google Wave，点击后面的链接查看 Google Wave 的各种介绍。需要 Wave 的邀请，可以在这里留下 Email。

https://blog.qizhen.xyz/2010/01/11/%E5%9C%A8google-wave%E4%B8%8A%E8%AE%A8%E8%AE%BAlabview%E9%97%AE%E9%A2%98

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2010年04月8日 15:25 编辑
至此，我的全部邀请以使用光了。不过大家还可以继续在这里发帖申请邀请，也许以前受过我邀请的人会再回来邀请别人:)

回复
锋
2010年04月6日 19:24 编辑
邀请我一下吧，谢谢！liufengsjzri@gmail.com

回复
立民
2010年04月6日 17:10 编辑
你好，给我也发一个，如果可能的话，谢了rlmchmrlm@gmail.com

回复
.
2010年03月30日 14:08 编辑
你好，如果还有邀请的话，请给我发一个邀请吧，ginger.7.chao@gmail.com，谢谢！！！

回复
Unknown
2010年03月28日 21:27 编辑
Google推出中国（大陆）的话对GoogleWave的使用有影响吗？

回复
whc
2010年03月9日 14:16 编辑
我的邮箱，hcwang.593@gmail.com谢谢

回复
Leon
2010年03月8日 11:32 编辑
邀请我一下吧，谢谢~~~ guanglin.ye@gmail.com

回复
婵
2010年03月2日 15:50 编辑
请邀请一下我吧，谢谢~beulah_xc@hotmail.com

回复
蛋
2010年02月20日 20:31 编辑
goofegg@gmail.com,邀请我一下啊。

回复
伟
2010年02月9日 13:38 编辑
给俺一个邀请吧，shmalcat@gmail.com, 谢谢！！

回复
金华
2010年02月3日 10:21 编辑
你好，经常看你的博客，受益匪浅，谢谢了。如果还有邀请名额，能否邀请我一下，jinhua.yan@gmail.com

回复
实
2010年01月21日 18:49 编辑
谢谢你的邀请，嘿嘿

回复
欣
2010年01月21日 17:15 编辑
我的邮箱：wangxin1944@hotmail.com多谢了！

回复
Qizhen
2010年01月21日 16:30 编辑
一一发过了:)

回复
实
2010年01月21日 09:27 编辑
你好，如果还有邀请的话，请给我发一个邀请吧，flycshi@gmail.com，谢谢！！！

回复
jia
2010年01月17日 09:48 编辑
如果您方便，能麻烦您也发一个邀请给我吗？Mail:lc880825@gmail.com ljgreat21@gmail.com

回复
Peter
2010年01月16日 23:42 编辑
如果您方便，能麻烦您也发一个邀请给我吗？Thanks in advance~Mail:dabao.gao@gmail.com

回复
磊
2010年01月15日 20:35 编辑
需要一个邀请，aegeann@gmail.com, 谢谢！！

回复
Qizhen
2010年01月15日 17:54 编辑
你的这个问题我不懂，建议发到论坛上询问一下。

回复
guo
2010年01月15日 14:24 编辑
最近用labview编写了一个程序，用到了matlab节点，写完程序之后打包生成了可以安装的exe程序，结果发现matlab command window无法打开，导致主程序“无响应”。网上给出的做法：将matscript.dll打包进来，但是我照做之后问题依旧存在，请问高手有没有解决的办法？

回复
可乐
2010年01月14日 21:48 编辑
yuchole@126.com 多谢

回复
lemon1900china
2010年01月14日 20:49 编辑
我的邮箱：zhengningzhou@gmail.com 谢谢你～

回复
帆
2010年01月14日 14:09 编辑
thewangfan在gmail.com 谢谢啊。。。

回复
Qizhen
2010年01月13日 13:33 编辑
回XiaoTian Zhang：http://ruanqizhen.spaces.live.com/blog/cns!5852D4F797C53FB6!4864.entry

回复
L
2010年01月13日 12:19 编辑
发给我一个吧andasheng#hotmail.com,谢谢

回复
XiaoTian
2010年01月13日 10:44 编辑
我想请教个问题：我设计了一个labview界面子VI，我想在主VI中多次调用该界面VI（同时执行，单独分配内存），并显示出多窗口，该如何设置？我尝试将子VI属性设置成可重入，仍无法解决。

回复
Unknown
2010年01月13日 06:29 编辑
我的email:xk104@hotmail.com

回复
LVBO
2010年01月12日 17:14 编辑
我的Email: zyanweb@163.com

回复
